### PR TITLE
Add meeting facillitation policy

### PR DESCRIPTION
**Pull request by mgattozzi**
In an effort to distribute team leadership more we developed ideas on
how to get others to run the meetings. This commit includes the policy
born out of that discussion.

### CODE REVIEW

**Comment by mgattozzi** [comment]
@ashleygwilliams @booyaa and @badboy here's the summary policy from today's meeting. Please poke at or correct anything if my understanding of what we talked about/action items for roles were/are incorrect

File: rfcs/meeting-facillitation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Meeting Facillitation
```
Comment (oe) [inline rfcs/meeting-facillitation.md]: fwiw facilitation only has one "l" in it

File: rfcs/meeting-facillitation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+-----
+
+Each meeting will have a facillitator and a note taker who sign up for the role for each meeting
+ahead of time. These roles are open to all Community Team members. The responsibilities for each are
+listed below.
+
+### Facillitator
+
+- On the Monday before a team meeting an agenda issue will be issued.
+  - This is done by copying the previous weeks agenda and removing unnecessary bits and then opening
```
Comment (oe) [inline rfcs/meeting-facillitation.md]: we could also have a template which seems like the less complicated solution to me

File: rfcs/meeting-facillitation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    up the issue
+  - The issue is then posted to IRC on #rust-community and mention all the team leads so they know
```
Comment (oe) [inline rfcs/meeting-facillitation.md]: `and all team leads are mentioned`

File: rfcs/meeting-facillitation.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    it has been posted
+  - If need be follow up a day later with leads who have not added their agenda items
+  - Add estimates for time as the agenda is updated
+- 10 minutes before the meeting ping everyone about it
+- Post the agenda and participation sheet at the beginning of the meeting
+- Run the meeting and sticking to the alloted time as much as possible
```
Comment (oe) [inline rfcs/meeting-facillitation.md]: `sticking` -> `stick`

File: rfcs/meeting-facillitation.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- Review participation sheet to make sure everyone was listed and observers/guests have been maked
```
Comment (oe) [inline rfcs/meeting-facillitation.md]: assuming it's supposed to say `marked` at the end?

**Review by ghost — COMMENTED** [review]
👍

File: rfcs/meeting-facillitation.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+-----
+
+Each meeting will have a facillitator and a note taker who sign up for the role for each meeting
+ahead of time. These roles are open to all Community Team members. The responsibilities for each are
+listed below.
+
+### Facillitator
+
+- On the Monday before a team meeting an agenda issue will be issued.
```
Comment (badboy) [inline rfcs/meeting-facillitation.md]: "issue will be issued" sounds so double. How about "an agenda issue should be created"

File: rfcs/meeting-facillitation.md (hunk 2 above)
Comment (badboy) [reply rfcs/meeting-facillitation.md]: See https://github.com/rust-community/content-team/blob/master/ISSUE_TEMPLATE/meeting_agenda.md

File: rfcs/meeting-facillitation.md (hunk 3 above)
Comment (badboy) [reply rfcs/meeting-facillitation.md]: Mention that team leads should fill in their respective topics

File: rfcs/meeting-facillitation.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  - The issue is then posted to IRC on #rust-community and mention all the team leads so they know
+    it has been posted
+  - If need be follow up a day later with leads who have not added their agenda items
+  - Add estimates for time as the agenda is updated
+- 10 minutes before the meeting ping everyone about it
+- Post the agenda and participation sheet at the beginning of the meeting
+- Run the meeting and sticking to the alloted time as much as possible
+- Review participation sheet to make sure everyone was listed and observers/guests have been maked
+
+### Note Taker
```
Comment (badboy) [inline rfcs/meeting-facillitation.md]: Add an additional empty line after this

File: rfcs/meeting-facillitation.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+- During the meeting keep track of what's discussed
+- On the agenda issue summarize what was discussed in the meeting and key action items/take aways
```
Comment (badboy) [inline rfcs/meeting-facillitation.md]: Should we encourage opening specific action item issues directly?

**Review by badboy — CHANGES_REQUESTED** [review]

File: rfcs/meeting-facillitation.md (hunk 2 above)
Comment (booyaa) [reply rfcs/meeting-facillitation.md]: Here's our own version https://github.com/rust-community/team/blob/master/ISSUE_TEMPLATE/meeting_agenda.md

**Comment by mgattozzi** [comment]
Hi all sorry I had to go away for like two weeks so I'll take a look and address these

**Pull request closed without merging**